docs/getting-started/overview.rst

```diff
@@ -1,6 +1,45 @@
+
+This edition of the .NET Framework is not cross-platform and you can run it only on Windows. Most of the consumers of this particular edition of the .NET Framework should be already know, if in name only. Windows Presentation Foundation or WPF is a framework for building desktop applications using specific technologies to make them compelling and rich. Windows Forms is already known as the bulwark of the LOB desktop applications from the early day of the .NET Framework. Of course, ASP.NET is Microsoft's main web stack, which is getting a major overhaul in the 5 release timeframe.
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, pay-as-you-go, cross-platform port of the .NET Framework. The roadmap currently in place covers support for three main operating systems, Linux, Windows and OS X. The community porters have made it work on FreeBSD as well. It is freely available on `GitHub <https://www.github.com/dotnet/coreclr>`_ and it is very easy to get for the supported systems and get going.
+
+Today, development of modern services and applications is very different from what was available a couple of years ago. Cloud services, application containers with Docker, strong client-side requirements and much more all change the way in which a modern developer may get about doing their everyday tasks. Many of these new developments mean that the developer wants to have a fully packaged environment that can be deployed in isolation, without affecting anything else, not to mention a way to include only those things that the application in question actually needs. 
+
+This is precisely why .NET Core came into the world. First, it is a completely componentized framework, meaning that everything is available as a package that you can opt-in. This includes the entire list of libraries that you would use in your application; you pull them in via packages and ship them as part of your application, and then update them as you see fit, without messing around with the centrally installated version. This makes it that much easier to deploy your apps in the cloud, Docker images or even hosting the runtime yourself if you're so inclined.
+
+.NET Native
+-----------
+Before we can explain what .NET Native is all about, we need to first cover some (very basic) ground on how .NET runtime currently executes code. The way .NET framework normally works is by JIT-ing IL (Intermediate Language) code. This happens at runtime. For the new generation of mobile and tablet applications, however, there are many scenarios where this is not really optimal, and where the developer would like to get to the actual code that is being run, without all of the JIT-ing and other things.
```

Comment:
We shouldn't tell readers what is and is not "very basic". That should be an exercise for the reader.

I'd start with what .NET Native is and then explain how it is different from other options. As a reader, I never appreciate reading an explanation of what something isn't first and then reading what it is.